### PR TITLE
Add the execution result of flake8 errors

### PR DESCRIPTION
**Pull request by yevpy**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,13 +1,19 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
+    return {"line": error["line_number"], "column": error["column_number"], "message": error["text"],
```
Comment (olena-hul) [inline app/main.py:2]: Fix flake8

**Review by olena-hul — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (yevpy) [reply app/main.py]: fixed

File: app/main.py [hunk 2]
```diff
@@ -1,13 +1,31 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
     pass
```
Comment (YukihiroSM) [inline app/main.py:9]: pass should be removed, it's useless in this case. (remove it in all functions)

File: app/main.py [hunk 3]
```diff
@@ -1,13 +1,31 @@
...
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
```
Comment (YukihiroSM) [inline app/main.py:12]: list of what? Add more descriptive typehint, please. Like list[SOME_DATATYPE]

**Review by YukihiroSM — CHANGES_REQUESTED** [review]

**Review by YukihiroSM — APPROVED** [review]
Looks good. Thanks for fixing issues.
For future you can also use even more descriptive type hints, using typing library. 
As example: 
`def format_linter_error(error: Dict[str, Any]) -> Dict[str, Any]`
But it depends on code readability, it's not always useful :)

**Pull request closed without merging**